### ISSUE TITLE
Researched on Google APIs

### ISSUE THREAD

**Issue by jekram** [Cloudkibo/ice#3]
Researched on Google APIs

Post links and finding here.

**Comment by jekram** [comment]
The Google API to retrieve billing information is in preview and is not currently enabled. However, it looks like it is very similar to AWS. Please see the link below:

https://support.google.com/cloud/answer/4524336?hl=en

billing2 is the bucket

**Comment by sojharo** [comment]
Salaam,

Is this bucket created on our google apps account.

**Comment by sojharo** [comment]
Please let me know if we need to go ahead with this using our google account

**Comment by jekram** [comment]
Yes. I have created an google account. However, I have currently blocked it as I to investigate this further.
